docs/tutorial/development-environment.md

```diff
@@ -0,0 +1,114 @@
+# Developer Environment
+
+Electron development is essentially Node.js development. To turn your operating
+system into an environment capable of building desktop apps with Electron,
+you will merely need Node.js, npm, a code editor of your choice, and a
+rudimentary understanding of your operating system's command line client.
+
+## Setting up macOS
+
+> Electron supports Mac OS X 10.9 (and all versions named macOS) and up. Apple does
+not allow running macOS in virtual machines unless the host computer is already
+an Apple computer, so if you find yourself in need of a Mac, consider using a
+cloud service that rents access to Macs (like [MacInCloud][macincloud] or
+[xcloud](https://xcloud.me)).
+
+First, install a recent version of Node.js. We recommend that you install
+either the latest `LTS` or `Current` version available. Visit
+[the Node.js download page][node-download] and select the `macOS Installer`.
+Once downloaded, execute the installer and let the installation wizard guide
+you through the installation.
```

Comment:
I would add a line in here somewhere about how homebrew is _not_ a good avenue for downloading node. I can't remember the exact details but it has something to do with the resultant binary being misnamed`nodejs`, or the install paths being wrong...